parsec/core/gui/workspaces_widget.py

```diff
@@ -119,56 +128,13 @@ def create_workspace_clicked(self):
         if not workspace_name:
             return
         try:
-            self.jobs_ctx.run(self.core.fs.workspace_create, os.path.join("/", workspace_name))
+            self.jobs_ctx.run(self.core.fs._user_fs.workspace_create, workspace_name)
```

Comment:
This `core.fs._user_fs.do_stuff` seems painful. Maybe we should start with adding a `user_fs` shortcut to `core` to prepare the removal of the `fs` interface.